### ISSUE TITLE
Some weird behavior when using small input u

### ISSUE THREAD

**Issue by Hhannzch** [tud-phi/soromox#42]
I'm using the code in [test](https://github.com/tud-phi/soromox/blob/main/examples/simulation/pcs/simulate_tendon_actuated_pcs.py) for TendonActuatedPCS. The parameters I'm using is (which is almost like the example):

      num_segments = 1
      rho = 1070 * jnp.ones(
          (num_segments,)
      )  # Volumetric density of Dragon Skin 20 [kg/m^3]
      params = {
          "p0": jnp.array([jnp.pi / 2, jnp.pi / 2, 0.0, 0.0, 0.0, 0.0]),  # Initial position and orientation
          "L": 0.25 * jnp.ones((num_segments,)),  # default: 1e-1
          "r": 0.025 * jnp.ones((num_segments,)),  # default: 2e-2
          "rho": rho,
          "g": jnp.array([0.0, 0.0, 0.0]),  # Gravity vector [m/s^2]
          "E": 2.5e5 * jnp.ones((num_segments,)),  # Elastic modulus [Pa]
          "G": 0.912e5 * jnp.ones((num_segments,)),  # Shear modulus [Pa]
      }
      params["D"] = 1e-3 * jnp.diag(
          (
              jnp.repeat(
                  jnp.array([[1e0, 1e0, 1e0, 1e3, 1e3, 1e3]]), num_segments, axis=0
              )
              * params["L"][:, None]
          ).flatten()
      )
      active_tendon_routing_params = {
              "ry": jnp.array([0.02, -0.02, 0.0, 0.0]),
              "rz": jnp.array([0.0, 0.0, 0.02, -0.02]),
              "my": jnp.array([0.0, 0.0, 0.0, 0.0]),
              "mz": jnp.array([0.0, 0.0, 0.0, 0.0]),
              "idx_seg_att": jnp.array([0, 0, 0, 0]),    
          }
  

    # ======================================================
    # Robot initialization
    # ======================================================

    robot = TendonActuatedPCS(
        num_segments=num_segments,
        params=params,
        active_tendon_routing_params=active_tendon_routing_params,
    )

    # =====================================================
    # Simulation upon time
    # =====================================================
    # Initial configuration

    q0 = jnp.repeat(
        jnp.array([0.0, 0.0,0.0,0.0,0.0, 0.0])[None, :],
        num_segments,
        axis=0,
    ).flatten()
    # Initial velocities
    qd0 = jnp.zeros_like(q0)

And I found with different input u, it shows different behavior but weird:

With u=[1.0, 0.0, 1.0, 0.0], the behavior is like:

https://github.com/user-attachments/assets/13112a96-60ee-49e5-9b82-56a166670179

With u=[0.05, 0.0, 0.05, 0.0], the behavior is like: (I think it is not correct)

https://github.com/user-attachments/assets/159e8255-a6be-435e-8a2f-163869a26f70

With u = [0.005, 0.0, 0.005, 0.0], the behavior is like: (I think it is good now)

https://github.com/user-attachments/assets/da87b9b7-4d85-498f-8007-1f2a36139704

Could I ask why it happens? Thank you very much!
